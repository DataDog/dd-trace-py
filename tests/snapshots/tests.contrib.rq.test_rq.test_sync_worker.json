[[
  {
    "name": "rq.queue.enqueue_job",
    "service": "rq",
    "resource": "tests.contrib.rq.jobs.job_add1",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "worker",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "component": "rq",
      "job.func_name": "tests.contrib.rq.jobs.job_add1",
      "job.id": "5d6d0f88-74a1-4041-8242-d2de0895e718",
      "language": "python",
      "queue.name": "q",
      "runtime-id": "dc862c6ff169400783e513c72f18a9f3",
      "span.kind": "producer"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 55795
    },
    "duration": 969388,
    "start": 1691167585412896937
  },
     {
       "name": "rq.worker.perform_job",
       "service": "rq-worker",
       "resource": "tests.contrib.rq.jobs.job_add1",
       "trace_id": 0,
       "span_id": 2,
       "parent_id": 1,
       "type": "worker",
       "error": 0,
       "meta": {
         "_dd.p.dm": "-0",
         "component": "rq",
         "job.id": "5d6d0f88-74a1-4041-8242-d2de0895e718",
         "job.origin": "q",
         "job.status": "finished",
         "language": "python",
         "runtime-id": "dc862c6ff169400783e513c72f18a9f3",
         "span.kind": "consumer",
         "traceparent": "00-000000000000000083b3755c33372db6-4d4ccf56bcccf344-01",
         "tracestate": "dd=s:1;t.dm:-0"
       },
       "metrics": {
         "_dd.top_level": 1,
         "_dd.tracer_kr": 1.0,
         "_sampling_priority_v1": 1,
         "process_id": 55795
       },
       "duration": 2045885,
       "start": 1691167585419796459
     },
        {
          "name": "rq.job.perform",
          "service": "rq-worker",
          "resource": "tests.contrib.rq.jobs.job_add1",
          "trace_id": 0,
          "span_id": 3,
          "parent_id": 2,
          "type": "",
          "error": 0,
          "meta": {
            "_dd.p.dm": "-0",
            "component": "rq",
            "job.id": "5d6d0f88-74a1-4041-8242-d2de0895e718"
          },
          "duration": 248154,
          "start": 1691167585420552114
        }]]
