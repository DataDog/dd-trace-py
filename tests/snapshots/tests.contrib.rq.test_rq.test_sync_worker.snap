[[{"name" "rq.queue.enqueue_job"
   "service" "rq"
   "resource" "tests.contrib.rq.jobs.job_add1"
   "type" "worker"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1630333148448191700
   "duration" 4758100
   "meta" {"runtime-id" "db58ccbfef044eeab2b2670b8b124068"
           "queue.name" "q"
           "job.id" "8ecd683e-ad34-44f3-bb24-bb5b1cd60475"
           "job.func_name" "tests.contrib.rq.jobs.job_add1"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 462
              "_dd.tracer_kr" 1.0}}
      {"name" "rq.worker.perform_job"
       "service" "rq-worker"
       "resource" "tests.contrib.rq.jobs.job_add1"
       "type" "worker"
       "error" 0
       "span_id" 1
       "trace_id" 0
       "parent_id" 0
       "start" 1630333148477384300
       "duration" 6117700
       "meta" {"runtime-id" "db58ccbfef044eeab2b2670b8b124068"
               "job.id" "8ecd683e-ad34-44f3-bb24-bb5b1cd60475"
               "job.status" "finished"
               "job.origin" "q"}
       "metrics" {"_sampling_priority_v1" 1
                  "system.pid" 462
                  "_dd.tracer_kr" 1.0}}
          {"name" "rq.job.perform"
           "service" "rq-worker"
           "resource" "tests.contrib.rq.jobs.job_add1"
           "error" 0
           "span_id" 2
           "trace_id" 0
           "parent_id" 1
           "start" 1630333148478759900
           "duration" 585100
           "meta" {"job.id" "8ecd683e-ad34-44f3-bb24-bb5b1cd60475"}}]]
